ofl/blakaink/DESCRIPTION.en_us.html

```diff
@@ -1 +1,18 @@
-TODO
+<p>
+    <em>Blaka</em> is an experimental typeface, enriching the gothic feeling of Black Letters by enhancing the
+    geometric features to create midgrounds with the Kufic style for the Arabic letterforms.
+</p>
+<p>
+    The aesthetical matching process between the two scripts relies on the features of being hand-drawn with a reed
+    pen. While Latin letterforms maintain sharp edges, Arabic relies on sharp edges and thick strokes to create overlaps
+    to replace a usual baseline in most letterforms, giving the Arabic letterforms contemporary features.
+</p>
+<p>
+    The font family comes in two styles, Blaka and Blaka Hollow, featuring an outlined version of the font. Blaka also
+    comes in two variants, the usual monochrome variants as well as an Ink variant <em>Blaka Ink</em>. The Ink variant
+    are color fonts where the colors are defined by the font itself. Support for color fonts is currently limited to few
+    applications like Google Chrome (version 98 or later).
+</p>
+<p>
+    To contribute, see <a href="https://github.com/Gue3bara/Blaka">github.com/Gue3bara/Blaka</a>.
```

Comment:
No final period on these lines please :) @felipesanches could be a FB check

Reply:
Why?